protocol/src/main/kotlin/tbdex/sdk/protocol/Validator.kt

```diff
@@ -70,4 +74,31 @@ object Validator {
       throw ValidatorException(message = "invalid payload", errors = validationMessages.map { it.message })
     }
   }
+
+  /**
+   * Validate a message against a predefined schema
+   * @param message The message data to validate.
+   * @throws Exception if validation fails, including a list of validation errors.
+   */
+  fun validateMessage(message: Message) {
+    val mapper = ObjectMapper()
```

Comment:
You can use the `jsonMapper` from `Json.kt` instead of reinstantiating

Reply:
done